core/src/main/java/nu/yona/server/properties/SecurityProperties.java

```diff
@@ -5,8 +5,9 @@
 	private int newDeviceRequestExpirationDays = 1;
 	private int passwordLength = 32;
 	private long bruteForceBlockMinutes = 60;
-	private long dosMemorySeconds = 300;
-	private boolean isDOSProtectionEnabled = false;
+	private long dosProtectionWindowSeconds = 300;
+	private boolean isDosProtectionEnabled = false;
+	private int maxCreateUserAttemptsPerTimeWindow = 1;
```

Comment:
Yes, but should you not also update the line in SecurityProperties.java from 1 to 2?